epochtalk/epochtalk
# Fix for angular code in meta tags

Updated the meta tags to not rely on angular code to set these tags.
Instead, they are filled through handlebars on initial rendering of the
public/index.html page through hapi. Any variables used by angular at a
later date are set in a script tag at the bottom of the page and pulled
in by angular as needed.

This also eliminates an extra call to the server on initial page load so
those routes and resources have been removed from both the server side
and client side.

TESTING PROCEDURES:
 * build as usual
 * open site and ensure that the title, keywords, descriptions, and favicon are pre-populated and not using angular variables.
